person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permision: true)
+    @id = Random.rand(1..10)
+    @name = name
+    @age = age
+    @parent_permission = parent_permision
+  end
+
+  def can_use_services?
+    if is_of_age? || @parent_permission
+      true
+    else
+      false
+    end
```

Comment:
- [ **OPTIONAL** ] I suggest you update the above code to something short such as ``is_of_age?() || @parent_permission`` as it is similar to the code selected above. This will make your code shorter and smarter.